commit f91f2a9879cc77db1f45f690f38f42698580416e
Author: Fenghua Yu <fenghua.yu@intel.com>
Date:   Wed Aug 28 16:34:00 2024 -0700

    dmaengine: idxd: Add a new DSA device ID for Granite Rapids-D platform
    
    A new DSA device ID, 0x11fb, is introduced for the Granite Rapids-D
    platform. Add the device ID to the IDXD driver.
    
    Since a potential security issue has been fixed on the new device, it's
    secure to assign the device to virtual machines, and therefore, the new
    device ID will not be added to the VFIO denylist. Additionally, the new
    device ID may be useful in identifying and addressing any other potential
    issues with this specific device in the future. The same is also applied
    to any other new DSA/IAA devices with new device IDs.
    
    Signed-off-by: Fenghua Yu <fenghua.yu@intel.com>
    Reviewed-by: Dave Jiang <dave.jiang@intel.com>
    Link: https://lore.kernel.org/r/20240828233401.186007-2-fenghua.yu@intel.com
    Signed-off-by: Vinod Koul <vkoul@kernel.org>

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index e388c8b1cbc2..ff99047dac44 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2706,6 +2706,7 @@
 #define PCI_DEVICE_ID_INTEL_82815_MC	0x1130
 #define PCI_DEVICE_ID_INTEL_82815_CGC	0x1132
 #define PCI_DEVICE_ID_INTEL_SST_TNG	0x119a
+#define PCI_DEVICE_ID_INTEL_DSA_GNRD	0x11fb
 #define PCI_DEVICE_ID_INTEL_82092AA_0	0x1221
 #define PCI_DEVICE_ID_INTEL_82437	0x122d
 #define PCI_DEVICE_ID_INTEL_82371FB_0	0x122e